openedx/features/course_experience/utils.py

```diff
@@ -281,13 +281,6 @@ def dates_banner_should_display(course_key, user):
     if not CourseEnrollment.is_enrolled(user, course_key):
         return False, False
 
-    # Don't display the banner for course staff
-    is_course_staff = bool(
-        user and course_overview and has_access(user, 'staff', course_overview, course_overview.id)
-    )
-    if is_course_staff:
-        return False, False
-
```

Comment:
Did you find out why we did that? Or was it a Nick decision?

Reply:
I think because we thought staff would always be able to just ignore due dates anyway and still submit problems, but turns out that doesn't seem to be the case.